code/game/turfs/paint_overlay.dm

```diff
@@ -36,10 +36,13 @@
 		paint_icon_state = paint_overlay_override_walls[paint_icon_state]
 	return paint_icon_state
 
-/turf/simulated/wall/mineral/silver/get_paint_state()
+/turf/simulated/wall/mineral/gold/get_paint_state()
+	return replacetext(icon_state,mineral,"silver")
```

Comment:
Yes. Most of the mineral walls have the exact same relief so I'm making economies in sprites by making all of them use the (colored to white) diamond walls, except the plasma and uranium walls which have some noticeable differences in their texture.

However #35257 added brand new textures for the gold and silver floors and walls, and since those once again have a fairly identical relief, I'm cutting in half the amount of sprites needed by making one use the other's "paint overlays".